File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/home/data/HomeGameProgress.java
package com.zwstudio.logicpuzzlesandroid.home.data;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

/**
 * Created by zwvista on 2016/09/29.
 */

@DatabaseTable
public class HomeGameProgress implements java.io.Serializable {
    @DatabaseField(generatedId = true)
    private int ID;
    @DatabaseField
    public String gameName = "LightenUp";
    @DatabaseField
    public boolean playMusic = true;
    @DatabaseField
    public boolean playSound = true;
}

File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/home/data/HomeGameProgress.kt
package com.zwstudio.logicpuzzlesandroid.home.data

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import java.util.*

open class HomeGameProgress : RealmObject() {
    @PrimaryKey
    var id = UUID.randomUUID().toString()
    var gameName = "LightenUp"
    var gameTitle = "Lighten Up"
    var playMusic = true
    var playSound = true
}